setup.py

```diff
@@ -132,7 +132,7 @@ def run(self):
 
 setup(
     name='azure-functions-python',
-    version='0.0.1',
+    version='2.0.0a2.dev0',
```

Comment:
we are still in preview. Lets just update the minor version for this

Reply:
Are you referring to the `dev` suffix?  It's a good idea to have it in between the tags so that we can better identify which version a user reporting an issue is running.

Reply:
I was referring to 2.0

Reply:
We will be releasing language -workers and host independently. We will not be able to keep their versions in sync.

Reply:
I'm not talking about keeping the *exact* version in sync, but just the major version as an epoch.  python-worker v2 is basically "python support for azure functions v 2.0".